templates/2025/navbar.html

```diff
@@ -72,11 +72,13 @@
                     >Team</a
                   >
                 </li>
-                <li class="dropdown-item">
+                <!-- <li class="dropdown-item">
                   <a class="nav-link g-px-0 g-color-black" href="/2025/coc/"
                     >Code of Conduct</a
                   >
-                </li>
+                </li> -->
+
+                <!-- Commented out some links in the About dropdown as they are catered for in the Attendee Guide dropdown now -->
```

Comment:
We could as well delete this(the comment plus the list items) as is dead code since is catered for under Attendee Guide